golang/pkg/dagent/dagent.go

```diff
@@ -50,3 +46,16 @@ func Serve(cfg *config.Configuration) {
 		EOF:        update.GRPCConnectionClosed,
 	})
 }
+
+func getAgentImageDate() string {
+	image, err := utils.GetOwnContainerImage(nil)
+	if err != nil {
+		if errors.Is(err, &utils.SelfUnknownError{}) {
+			return "-"
+		}
+
+		log.Panic().Err(err).Msg("failed to get own container")
```

Comment:
We talked in person with @nandor-magyar, we can use upper case for the first character in the sentence. Error message got the lowercase version.